src/services/search.service.ts

```diff
@@ -10,21 +10,26 @@ import { CipherType } from '../enums/cipherType';
 import { FieldType } from '../enums/fieldType';
 import { UriMatchType } from '../enums/uriMatchType';
 import { SendView } from '../models/view/sendView';
+import { I18nService } from '../abstractions';
```

Comment:
Imports are grouped and should be in alphabetical order within each group. Please move this import to the other `abstractions` group, in the proper order.